source/GameAction.cpp

```diff
@@ -214,6 +215,16 @@ void GameAction::LoadSingle(const DataNode &child, const string &missionName)
 			GameData::Missions().Get(toFail);
 		}
 	}
+	else if(key == "relocate")
+	{
+		for(const DataNode &grand : child)
+		{
+			if(grand.Token(0) == "planet" && grand.Size() > 1)
+				teleportPlanet = GameData::Planets().Get(grand.Token(1));
```

Comment:
This perhaps ought to work like the `destination` node of a mission.
`destination <planet name>` will link directly to the named planet.
But, the node can also have children which will be used to create a LocationFilter from which a specific planet is selected.
LocationFilter support isn't necessary here, but I would prefer `destination` be the syntax to better match the usage in missions.

Reply:
Sounds good, but how will we then support relocation to space in a system? That option is not in this PR, but I expect we do want to add it later.
(Maybe we should support systems as mission destinations by then?)

Reply:
That's a good point. I suppose it should remain as `planet` in that case.

Reply:
I agree with that